modules/Core/src/main/java/org/terasology/core/world/generator/facetProviders/PerlinBaseSurfaceProvider.java

```diff
@@ -55,7 +55,7 @@ public void process(GeneratingRegion region) {
         float[] noise = surfaceNoise.noise(processRegion);
 
         for (int i = 0; i < noise.length; ++i) {
-            noise[i] = seaLevel + seaLevel * ((noise[i] + 1f) / 2f);
+            noise[i] = seaLevel + seaLevel * ((noise[i] * 2.11f + 1f) / 2f);
```

Comment:
Hrm,  I may be answering my own question.  msteiger,  maybe just confirm...

The previous implementation of the brownian noise was likely returning -2.11 to +2.11?